arangod/RocksDBEngine/RocksDBPersistedLog.cpp

```diff
@@ -33,6 +33,8 @@
 #include "RocksDBKey.h"
 #include "RocksDBPersistedLog.h"
 #include "RocksDBValue.h"
+#include "Scheduler/Scheduler.h"
+#include "Scheduler/SchedulerFeature.h"
```

Comment:
Why do we need these header files? for function2?

Reply:
We don't need them, I'll remove them.
```suggestion
#include "Scheduler/Scheduler.h"
#include "Scheduler/SchedulerFeature.h"
```